docs/index.rst

```diff
@@ -12,4 +12,5 @@ free and open-source solar data analysis environment for Python.
   generated/gallery/index
   dev_guide/index
   whatsnew/index
+  stability
```

Comment:
some unhelpful comment about too many top level links.

Reply:
But where should I put it? In the dev guide?